AWSPinpoint/AWSPinpointEvent.m

```diff
@@ -115,12 +110,12 @@ - (void)addAttribute:(NSString *)theValue forKey:(NSString *)theKey {
     if(!theKey) return;
     
     @synchronized(self.attributes) {
-        if(self.currentNumOfAttributesAndMetrics < MAX_NUM_OF_METRICS_AND_ATTRIBUTES) {
+        if(self.attributes.count < MAX_NUM_OF_METRICS_AND_ATTRIBUTES) {
```

Comment:
I think the correct quota check would be the same as before BUT the comparison should be <=. Also the same warning applies.

Reply:
Thanks for the clarification I understand now why the condition is < and not <=.